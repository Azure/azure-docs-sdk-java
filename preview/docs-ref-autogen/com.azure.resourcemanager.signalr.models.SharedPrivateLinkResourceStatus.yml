### YamlMime:JavaType
uid: "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus"
fullName: "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus"
name: "SharedPrivateLinkResourceStatus"
nameWithType: "SharedPrivateLinkResourceStatus"
summary: "Defines values for SharedPrivateLinkResourceStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.azure.core.util.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.azure.core.util.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.azure.core.util.ExpandableStringEnum.equals(java.lang.Object)"
- "com.azure.core.util.ExpandableStringEnum.hashCode()"
- "com.azure.core.util.ExpandableStringEnum.toString()"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class SharedPrivateLinkResourceStatus extends ExpandableStringEnum<SharedPrivateLinkResourceStatus>"
constructors:
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus.SharedPrivateLinkResourceStatus()"
fields:
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus.APPROVED"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus.DISCONNECTED"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus.PENDING"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus.REJECTED"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus.TIMEOUT"
methods:
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus.fromString(java.lang.String)"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus.values()"
type: "class"
metadata: {}
package: "com.azure.resourcemanager.signalr.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-signalr:1.0.0-beta.1
