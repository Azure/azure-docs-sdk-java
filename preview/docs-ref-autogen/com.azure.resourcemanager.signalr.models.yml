### YamlMime:JavaPackage
uid: "com.azure.resourcemanager.signalr.models"
fullName: "com.azure.resourcemanager.signalr.models"
name: "com.azure.resourcemanager.signalr.models"
summary: "Package containing the data models for SignalRManagementClient. REST API for Azure SignalR Service."
classes:
- "com.azure.resourcemanager.signalr.models.AclAction"
- "com.azure.resourcemanager.signalr.models.Dimension"
- "com.azure.resourcemanager.signalr.models.FeatureFlags"
- "com.azure.resourcemanager.signalr.models.KeyType"
- "com.azure.resourcemanager.signalr.models.LogSpecification"
- "com.azure.resourcemanager.signalr.models.ManagedIdentity"
- "com.azure.resourcemanager.signalr.models.ManagedIdentitySettings"
- "com.azure.resourcemanager.signalr.models.ManagedIdentityType"
- "com.azure.resourcemanager.signalr.models.MetricSpecification"
- "com.azure.resourcemanager.signalr.models.NameAvailabilityParameters"
- "com.azure.resourcemanager.signalr.models.NetworkAcl"
- "com.azure.resourcemanager.signalr.models.OperationDisplay"
- "com.azure.resourcemanager.signalr.models.OperationList"
- "com.azure.resourcemanager.signalr.models.OperationProperties"
- "com.azure.resourcemanager.signalr.models.PrivateEndpoint"
- "com.azure.resourcemanager.signalr.models.PrivateEndpointAcl"
- "com.azure.resourcemanager.signalr.models.PrivateEndpointConnectionList"
- "com.azure.resourcemanager.signalr.models.PrivateLinkResourceList"
- "com.azure.resourcemanager.signalr.models.PrivateLinkServiceConnectionState"
- "com.azure.resourcemanager.signalr.models.PrivateLinkServiceConnectionStatus"
- "com.azure.resourcemanager.signalr.models.ProvisioningState"
- "com.azure.resourcemanager.signalr.models.RegenerateKeyParameters"
- "com.azure.resourcemanager.signalr.models.ResourceSku"
- "com.azure.resourcemanager.signalr.models.ServerlessUpstreamSettings"
- "com.azure.resourcemanager.signalr.models.ServiceKind"
- "com.azure.resourcemanager.signalr.models.ServiceSpecification"
- "com.azure.resourcemanager.signalr.models.ShareablePrivateLinkResourceProperties"
- "com.azure.resourcemanager.signalr.models.ShareablePrivateLinkResourceType"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceList"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus"
- "com.azure.resourcemanager.signalr.models.SignalRCorsSettings"
- "com.azure.resourcemanager.signalr.models.SignalRFeature"
- "com.azure.resourcemanager.signalr.models.SignalRNetworkACLs"
- "com.azure.resourcemanager.signalr.models.SignalRRequestType"
- "com.azure.resourcemanager.signalr.models.SignalRResourceList"
- "com.azure.resourcemanager.signalr.models.SignalRSkuTier"
- "com.azure.resourcemanager.signalr.models.SignalRTlsSettings"
- "com.azure.resourcemanager.signalr.models.SignalRUsageList"
- "com.azure.resourcemanager.signalr.models.SignalRUsageName"
- "com.azure.resourcemanager.signalr.models.UpstreamAuthSettings"
- "com.azure.resourcemanager.signalr.models.UpstreamAuthType"
- "com.azure.resourcemanager.signalr.models.UpstreamTemplate"
- "com.azure.resourcemanager.signalr.models.UserAssignedIdentityProperty"
interfaces:
- "com.azure.resourcemanager.signalr.models.NameAvailability"
- "com.azure.resourcemanager.signalr.models.Operation"
- "com.azure.resourcemanager.signalr.models.Operations"
- "com.azure.resourcemanager.signalr.models.PrivateEndpointConnection"
- "com.azure.resourcemanager.signalr.models.PrivateLinkResource"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResource"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResource.Definition"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResource.DefinitionStages"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResource.DefinitionStages.Blank"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResource.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResource.DefinitionStages.WithGroupId"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResource.DefinitionStages.WithParentResource"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResource.DefinitionStages.WithPrivateLinkResourceId"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResource.DefinitionStages.WithRequestMessage"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResource.Update"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResource.UpdateStages"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResource.UpdateStages.WithGroupId"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResource.UpdateStages.WithPrivateLinkResourceId"
- "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResource.UpdateStages.WithRequestMessage"
- "com.azure.resourcemanager.signalr.models.SignalRKeys"
- "com.azure.resourcemanager.signalr.models.SignalRPrivateEndpointConnections"
- "com.azure.resourcemanager.signalr.models.SignalRPrivateLinkResources"
- "com.azure.resourcemanager.signalr.models.SignalRResource"
- "com.azure.resourcemanager.signalr.models.SignalRResource.Definition"
- "com.azure.resourcemanager.signalr.models.SignalRResource.DefinitionStages"
- "com.azure.resourcemanager.signalr.models.SignalRResource.DefinitionStages.Blank"
- "com.azure.resourcemanager.signalr.models.SignalRResource.DefinitionStages.WithCors"
- "com.azure.resourcemanager.signalr.models.SignalRResource.DefinitionStages.WithCreate"
- "com.azure.resourcemanager.signalr.models.SignalRResource.DefinitionStages.WithFeatures"
- "com.azure.resourcemanager.signalr.models.SignalRResource.DefinitionStages.WithIdentity"
- "com.azure.resourcemanager.signalr.models.SignalRResource.DefinitionStages.WithKind"
- "com.azure.resourcemanager.signalr.models.SignalRResource.DefinitionStages.WithLocation"
- "com.azure.resourcemanager.signalr.models.SignalRResource.DefinitionStages.WithNetworkACLs"
- "com.azure.resourcemanager.signalr.models.SignalRResource.DefinitionStages.WithResourceGroup"
- "com.azure.resourcemanager.signalr.models.SignalRResource.DefinitionStages.WithSku"
- "com.azure.resourcemanager.signalr.models.SignalRResource.DefinitionStages.WithTags"
- "com.azure.resourcemanager.signalr.models.SignalRResource.DefinitionStages.WithTls"
- "com.azure.resourcemanager.signalr.models.SignalRResource.DefinitionStages.WithUpstream"
- "com.azure.resourcemanager.signalr.models.SignalRResource.Update"
- "com.azure.resourcemanager.signalr.models.SignalRResource.UpdateStages"
- "com.azure.resourcemanager.signalr.models.SignalRResource.UpdateStages.WithCors"
- "com.azure.resourcemanager.signalr.models.SignalRResource.UpdateStages.WithFeatures"
- "com.azure.resourcemanager.signalr.models.SignalRResource.UpdateStages.WithIdentity"
- "com.azure.resourcemanager.signalr.models.SignalRResource.UpdateStages.WithNetworkACLs"
- "com.azure.resourcemanager.signalr.models.SignalRResource.UpdateStages.WithSku"
- "com.azure.resourcemanager.signalr.models.SignalRResource.UpdateStages.WithTags"
- "com.azure.resourcemanager.signalr.models.SignalRResource.UpdateStages.WithTls"
- "com.azure.resourcemanager.signalr.models.SignalRResource.UpdateStages.WithUpstream"
- "com.azure.resourcemanager.signalr.models.SignalRSharedPrivateLinkResources"
- "com.azure.resourcemanager.signalr.models.SignalRUsage"
- "com.azure.resourcemanager.signalr.models.SignalRs"
- "com.azure.resourcemanager.signalr.models.Usages"
metadata: {}
package: "com.azure.resourcemanager.signalr.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-signalr:1.0.0-beta.1
