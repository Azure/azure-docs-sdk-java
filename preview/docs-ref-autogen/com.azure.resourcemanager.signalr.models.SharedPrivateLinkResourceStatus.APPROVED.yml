### YamlMime:JavaMember
uid: "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus.APPROVED*"
fullName: "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus.APPROVED"
name: "APPROVED"
nameWithType: "SharedPrivateLinkResourceStatus.APPROVED"
members:
- uid: "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus.APPROVED"
  fullName: "com.azure.resourcemanager.signalr.models.SharedPrivateLinkResourceStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "SharedPrivateLinkResourceStatus.APPROVED"
  summary: "Static value Approved for SharedPrivateLinkResourceStatus."
  syntax: "public static final SharedPrivateLinkResourceStatus APPROVED"
type: "field"
metadata: {}
package: "com.azure.resourcemanager.signalr.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-signalr:1.0.0-beta.1
